modules/installation-uninstall-vsphere.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/uninstalling-cluster-vsphere.adoc
+
+[id="installation-uninstall-vsphere_{context}"]
+= Removing a cluster from vSphere
+
+You can remove a cluster that you installed on vSphere.
+
+.Procedure
```

Comment:
Should we mention anything about removing whatever networking infrastructure was established to support the cluster? Or is it assumed that the user will know that those should be undone since they created them?